### PR TITLE
SelectControl: Add onChange, onBlur and onFocus to storybook actions

### PR DESCRIPTION
**Pull request by walbo**
## What?
Log `onChange`, `onBlur` and `onFocus` props to the Actions screen in Storybook.

## Why?
To see what `onChange`, `onBlur` and `onFocus` does.

## How?
Update `SelectControl` storybook to log all props stating with `on` in the Actions screen.

## Testing Instructions
1. Fire up the storybook
2. Select the `SelectControl` story and change the value.
3. Confirm `onChange`, `onBlur` and `onFocus` was logged in the Actions screen.

## Screenshots or screencast 
select-control-actions

### CODE REVIEW

File: packages/components/src/select-control/stories/index.tsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ const meta: ComponentMeta< typeof SelectControl > = {
 		prefix: { control: { type: 'text' } },
 		suffix: { control: { type: 'text' } },
 		value: { control: { type: null } },
+		onChange: { action: 'onChange' },
```
Comment (ciampo) [inline packages/components/src/select-control/stories/index.tsx]: An alternative that we could look into adopting across the package is to use a regex to mark as actions all `on*` props (see for example [`FocalPointPicker`'s Storybook parameters](https://github.com/WordPress/gutenberg/blob/4efdb495b2ce535a03fb4765797fd0af1b5c9ec8/packages/components/src/focal-point-picker/stories/index.tsx))

**Review by ciampo — APPROVED** [review]
🚀  Thank you!

File: packages/components/src/select-control/stories/index.tsx (hunk 1 above)
Comment (walbo) [reply packages/components/src/select-control/stories/index.tsx]: Thanks. Didn't know you could do that. Would be great to get `onBlur` and `onFocus` as well.  I'll update the PR later today.

Comment (walbo) [reply packages/components/src/select-control/stories/index.tsx]: Updated the PR to match all `on*` using regex.

**Review by ciampo — APPROVED** [review]

**Pull request merged**